### PR TITLE
Title Component

### PR DESCRIPTION
**Pull request by moralesbang**
## Summary

- Added title's component
- Setted `Roboto` as global font

## Screenshots

Screen Shot 2019-07-15 at 12 36 58 PM

## Trello Card

https://trello.com/c/OOGNKOap

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="title">
+    <img src="images/triangle.svg" alt="triangle's logo" class="title-decoration">
```
Comment (henryzarza) [inline maquetado/index.html]: What do you think about of use this image as background?

File: maquetado/index.html [hunk 2]
```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="title">
```
Comment (henryzarza) [inline maquetado/index.html]: You don't need this, you can define a structure more simple

File: maquetado/scss/application.scss [hunk 3]
```diff
@@ -1,2 +1,3 @@
 @import 'equalizer';
 @import 'styles';
+@import 'titles';
```
Comment (henryzarza) [inline maquetado/scss/application.scss]:
```suggestion
@import 'title';
```
Change the folder name for singular

File: maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+.title {
+  align-items: center;
+  display: flex;
+
+  &-decoration {
+    height: 90px;
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: Use `title-decoration-height: 90px;` variable defined in the [card](https://trello.com/c/qMZlqheW/11-setup-del-proyecto)

File: maquetado/scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    width: 70px;
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: The same here but with `title-decoration-width: 70px;`

File: maquetado/scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  }
+
+  &-text {
+    left: -18px;
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: Why this?

File: maquetado/scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    position: relative;
+  }
+}
+
+.text-big {
+  font-size: 24px;
```
Comment (henryzarza) [inline maquetado/scss/titles.scss]: Use `text-big` variable

File: maquetado/scss/title.scss [hunk 8]
```diff
@@ -0,0 +1,16 @@
+.title {
+  align-items: center;
+  display: flex;
+  height: $text-decoration-height;
+  font-size: $text-big;
```
Comment (henryzarza) [inline maquetado/scss/title.scss]: Remember order by alphabetically. Check this [styles guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#properties-order)

File: maquetado/scss/title.scss [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+  &::before {
+    content: '';
+    background: url('images/triangle.svg') no-repeat;
```
Comment (henryzarza) [inline maquetado/scss/title.scss]: The same about of order

File: maquetado/scss/title.scss [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+    height: 100%;
+    left: 20px;
+    position: relative;
```
Comment (henryzarza) [inline maquetado/scss/title.scss]: Do you need this property?

Comment (moralesbang) [reply maquetado/scss/title.scss]: `position: relative` ?

**Pull request merged**